packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -234,8 +234,7 @@ export default function ShadowsEditPanel() {
 						>
 							<FlexItem>
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> shadows --> add shadow and rename it, a modal would be opened and check for `cancel` button.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="shadow-rename-cancel-before" src="https://github.com/user-attachments/assets/384ac330-5f64-4367-b481-b2497f7bfa62"> | <img width="625" alt="shadow-rename-cancel-after" src="https://github.com/user-attachments/assets/fc34cfb5-8477-4d52-bd39-a7fa68a3dddd"> |